test/jdk/com/sun/java/swing/plaf/DesktopPropertyResetPendingFlagTest.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8367376
+ * @summary DesktopProperty never reset pending status to process new updates
+ * @modules java.desktop/sun.swing.plaf
+ * @run main DesktopPropertyResetPendingFlagTest
+ * @key headful
+ */
```

Comment:
```suggestion
/*
 * @test
 * @bug 8367376
 * @key headful
 * @summary DesktopProperty never reset pending status to process new updates
 * @modules java.desktop/sun.swing.plaf
 * @run main DesktopPropertyResetPendingFlagTest
 */
```
It doesn't really matter, yet usually `@key` comes after the `@bug` tag.